fxslack/module_test.go

```diff
@@ -0,0 +1,173 @@
+package fxslack_test
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/ankorstore/yokai-contrib/fxslack"
+	"github.com/ankorstore/yokai/config"
+	"github.com/ankorstore/yokai/fxconfig"
+	"github.com/slack-go/slack"
+	"github.com/slack-go/slack/slacktest"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/fx"
+	"go.uber.org/fx/fxtest"
+)
+
+func TestFxSlackModule(t *testing.T) {
+	app := fxtest.New(
+		t,
+		fx.NopLogger,
+		fxslack.FxSlackModule,
+		fxconfig.FxConfigModule,
+	)
+
+	app.RequireStart()
+	assert.NoError(t, app.Err(), "failed to start the Fx application")
+
+	app.RequireStop()
+	assert.NoError(t, app.Err(), "failed to stop the Fx application")
+}
+
+func TestFxSlackClient(t *testing.T) {
+	t.Setenv("APP_ENV", "dev")
```

Comment:
you use `config.AppEnvTest` in the test below, we should have consistency, wdyt ? (string or const, I let you choose)

And if you decide to run your tests all in env `test`, you can get rid of the `testdata/configs/config.dev.yaml`, since it does not contain much and you can drive Yokai test apps via env vars

Reply:
I'll keep both so that I can actually test the normal slack client